app/survey_config/survey_config.py

```diff
@@ -26,6 +28,16 @@ class SurveyConfig:
     account_service_surveys_path: Optional[str] = None
     title_logo: Optional[str] = None
     title_logo_alt: Optional[str] = None
+    contact_us_url: tuple[dict[str, Any]] = (
+        Link(lazy_gettext("Contact us"), f"{base_url}/contact-us/").__dict__,
+    )
+    cookie_settings_url: Optional[str] = f"{base_url}/cookies"
+    privacy_and_data_protection: tuple[dict[str, Any]] = (
```

Comment:
Perhaps should be `privacy_and_data_protection_url` for consistency

Reply:
Added, thanks!